samples/Sentry.Samples.Console.Basic/Program.cs

```diff
@@ -1,6 +1,6 @@
 using Sentry;
 
-using (SentrySdk.Init("https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537"))
+using (SentrySdk.Init(o=> { o.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537"; o.Debug = true; o.DiagnosticLevel = SentryLevel.Debug; }))
```

Comment:
It's debug by default:

Is it better to break lines here instead? Something like:

```suggestion
using (SentrySdk.Init(o=> 
    {
        o.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537"
        o.Debug = true;
     }))
```